noir/noir-repo/compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_block.rs

```diff
@@ -831,14 +831,17 @@ impl<'block> BrilligBlock<'block> {
             _ => unreachable!("ICE: array set on non-array"),
         };
 
-        let one = self.brillig_context.make_usize_constant(1_usize.into());
+        // Here we want to compare the reference count against 1.
+        // I don't think we have the bit_size for the reference count available, so I'm hardcoding it.
+        let bit_size = usize::BITS;
```

Comment:
I believe that this is the sane as BRILLIG_MEMORY_ADDRESSING_BIT_SIZE?

Reply:
Yes we should use BRILLIG_MEMORY_ADDRESSING_BIT_SIZE, rust's usize is the pointer size of the target native machine we're compiling to. We can just use the old `let one = self.brillig_context.make_usize_constant(1_usize.into());` and keep the change in binary_instruction:
`BrilligBinaryOp::Integer { op: BinaryIntOp::Equals, bit_size: BRILLIG_MEMORY_ADDRESSING_BIT_SIZE },` or use `self.brillig_context.memory_op` that already uses the brillig's usize

Reply:
Done with memory_op!